### PR TITLE
[stable/nginx-ingress] Upgrade to 0.17.1

### PR DESCRIPTION
**Pull request by jpds**
**What this PR does / why we need it**: Upgrade to 0.17.1.

### CODE REVIEW

**Comment by davidkarlsen** [comment]
Also see https://github.com/helm/charts/pull/7001

**Comment by jpds** [comment]
I have no idea why the `pull-charts-e2e` thing is failing, it just has a generic error.

**Comment by jpds** [comment]
@sebastien-prudhomme Done.

File: stable/nginx-ingress/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ controller:
   name: controller
   image:
     repository: quay.io/kubernetes-ingress-controller/nginx-ingress-controller
-    tag: "0.15.0"
+    tag: "0.17.1"
```
Comment (davidkarlsen) [inline stable/nginx-ingress/values.yaml:8]: Can you update the readme to match the values.yaml?

File: stable/nginx-ingress/values.yaml [hunk 2]
```diff
@@ -291,7 +291,7 @@ defaultBackend:
   name: default-backend
   image:
     repository: k8s.gcr.io/defaultbackend
-    tag: "1.3"
+    tag: "1.4"
```
Comment (davidkarlsen) [inline stable/nginx-ingress/values.yaml:294]: same here

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by davidkarlsen** [comment]
Reference to docs explaining why securityContext needs update: https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.16.2

File: stable/nginx-ingress/values.yaml (hunk 1 above)
Comment (jpds) [reply stable/nginx-ingress/values.yaml:8]: Done.

**Pull request merged**